src/planner/binder/expression/bind_columnref_expression.cpp

```diff
@@ -388,4 +388,10 @@ BindResult ExpressionBinder::BindExpression(ColumnRefExpression &colref_p, idx_t
 	return result;
 }
 
+bool ExpressionBinder::QualifyColumnAlias(const ColumnRefExpression &colref) {
+	// Only BaseSelectBinder will have a valid col alias map,
+	// otherwise just return false
```

Comment:
Shouldn't we use the parent binder in this case?

Reply:
I think only `BaseSelectBinder` has the alias_map we need to check
There is no alias_map data member in `ExpressionBinder` (parent class)